### ISSUE TITLE
[a11y Support & Content Support] VA Notify - Email Template Accessibility

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#85074]
# CAIA A11y Support
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.
## Menu Selection
_**Work necessary for this ticket to be completed (select all that apply):**_
- [x] [Slack](https://dsva.slack.com/archives/C03MP8MEPCN/p1717516302616179) Follow-Up Ticket
- [x] Tracking for CAIA Support (A11Y and Content)

## CAIA Assignee(s)
- `Ticket A11y Lead` @coforma-jamie 
- `Ticket Content Lead` @laurwill 
- `A11y Ticket Support` @sara-amanda 
## Team Info
<details><summary>Toggle to view team info</summary>

- **Team name:** VA Notify
- **OCTO product owner:** `OCTO product owner`
- **Product name:** `Product name`
- **Product manager:** `Product manager`
- **Slack channel:** [#va-notify-public](https://dsva.slack.com/archives/C010R6AUPHT)
- **Dedicated content writer on your team (if you have one):** `writer`
- **Dedicated a11y specialist on your team (if you have one):** `a11y`

</details>

## Description

A11Y support reviewing potential screen reader impact for @laurwill via Slack to benefit VA Notify Team.

<details><summary>Toggle for additional information and images</summary>

### VA Notify Email Template: Appointment Reminders

**[From Laura 6/4/2024](https://dsva.slack.com/archives/C03MP8MEPCN/p1717516302616179):** 

> I'm working on a VA Notify email template for appointment reminders, and had a couple accessibility Qs if anyone has a moment. Screenshot in :thread: , and I can also send a test email.

> VA Notify doesn't allow headers below H2, and this email has an H2 followed by a list of appointments with several lines of related data. Bulleted list looked funky in the template, so I tried bold on the first line of each appointment. Is there a more accessible way to approach this, or is this okay?

> VA Notify team suggested we spell out all URLs rather than hyperlinking text, so it's easy to confirm where link is going. The URLs themselves become hyperlinked if we include https://www/ . Is this okay for screenreaders?

> So this suggestion to spell out urls is due to research findings that people are suspicious of clicking links in emails.

### Details
_What details are necessary for understanding the specific work or request tracked by this issue?_

- H1 and H2 levels only (no H3 permitted by VA Notify)
- Bold text being used
- Bulleted list appearance issues in template
- URLs spelled out vs. hyperlinking

### Template Image

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/124186314/3292606d-f6a2-4cb0-939c-fa2d2ea413c3)

</details>

### Semantic Value Needed
- Screen readers won’t hear that its bold / emphasized. those bolds won’t have any semantic value. it’ll be something like
    - `Heading level 2, Your upcoming VA appointments.`
    - `In person appointment, May 13 2024 at 4pm MT. For questions…`
- People navigating by heading will have access to the following. “missing” headings in italics
    - `H1` VA appointment reminder
        - `H2` Your upcoming VA appointments
            - `H3` In-person
            - `H3` Phone
            - `H3` Video….
    - `H2` How to manage your appointments
    - `H2` How to prepare for your appointments
    - `H2` How to change your notification preferences
    - `H2` Need help?

> [!TIP]
> #### Recommendation
> Which honestly, I think is ok? It’d be cool if the H3s were present, but it should be clear for someone whats in that section. Hitting DOWN a few more times is probably fine. Long way to say, that should be ok IMO

#### Spelling out URLs
The issue with spelling them out is that the assistive tech will read the link out loud and thats a lot of cruft to wade through...
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/124186314/1d1d6cb0-ad50-4788-a2d7-b5dd1da07360)

#### Comparing Hyperlinks
| V1 | V2 |
|--------|--------|
| `link, Manage your VA appointments online` | `Manage your VA appointments online at link, h t t p colon slash slash w w w dot [va.gov](http://va.gov/) slash health dash care slash schedule dash view dash va dash appointments` |

> [!TIP]
> #### Recommendation
> - I would recommend changing the copy to “`[link]Manage your VA appointments on VA.gov [/link]`” if they want folks to be able to affirm that its going to the VA website.
> - Don't have the full URL as visual text at all. if the concern is “how does the user know that they’re going to the VA website?“, then you can say “do X on VA.gov” as the link.
> - Beware of linking the full URL for more clicks. The URL can be hyperlinked to anything, so it isn't really the best way to indicate a safe URL. 

### AT Users
- For AT users (at least for screen reader and voice to speech), a typical call-to-action text link is better UX than a spelled-out URL. 
- So something like` "Manage your appointments on VA.gov"`

### Phishing Concerns
- For people who are concerned about phishing and want to make sure they're going to a legit `[VA.gov](http://va.gov/) `link, a spelled-out URL that is not hyperlinked might be the best solution? 
- That way, they can copy/paste the URL into their browser without accidentally clicking on it (since a hyperlinked URL is still a phishing concern).
- So that might look something like this: `[Manage your appointments on VA.gov](https://www.va.gov/health-care/schedule-view-va-appointments/)
(https://www.va.gov/health-care/schedule-view-va-appointments/)`

### Template Behavior with Links 

- The VA Notify template automatically hyperlinks a full spelled-out URL.
    - So for that approach, we might need to remove the` https://www/.` and start the URL with` [va.gov](http://va.gov/).` 
    - I know we had included the https://www/. in the past because of an issue with incomplete URLs not loading if you're on the VA network, but this works fine for me in all the browsers I've tried on the network.

## Definition of Done
```[tasklist]
### A11y Tasks
- [x] Internal request[ CAIA Content to CAIA A11Ys via Slack ](https://dsva.slack.com/archives/C03MP8MEPCN/p1717516302616179)on `6/4/2024`
- [x] @sara-amanda created ticket to track support work and connect to current work
- [x] @coforma-jamie provided feedback to @laurwill  `6/4/2024`
```

```[tasklist]
### Content Tasks
- [x] Inquiry in caia-internal DSVA channel by @laurwill 6/4/2024
- [x] `Outreach` to the VFS team via Slack to be done by Content Team. 
- [x] [Thread being started with VA Notify Team](https://dsva.slack.com/archives/C010R6AUPHT/p1717530522541419), CAIA Content and CAIA A11Ys c/o @laurwill 
- [x] Waiting on feedback from approach in VA Notify Slack Channel
- [ ] Determine final feedback with VA Notify CAIA A11Ys and CAIA Content
```

```[tasklist]
## Ticket Updates (CAIA Internal)
- [ ] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_ `Need Ticket Number to Connect` @strelichl or @laurwill 
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [x] _Label date in the `Open Date` field_
- [x] _Label with `Estimate` (level of effort expected for this ticket)_
- [x] _Add `Assignee(s)` name(s) to ticket_
- [x] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [x] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in `Closed  Date`_
```
